_sass/components/_identifier.scss

```diff
@@ -0,0 +1,140 @@
+.contact-bar {
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.usa-identifier {
+  background-color: $theme-identifier-background-color;
+  color: $theme-identifier-color;
+  padding-top: 1rem;
+  padding-bottom: 3rem;
+
+  font-family: $font-sans;
+  font-weight: 400;
+  font-size: 1.5rem;
```

Comment:
We're setting font-size to 1.5rem here, but other classes like `.usa-identifier__identity-domain` show 1.6rem.

Do we need that small difference?

Reply:
Visually, it makes it look closer to the identifier component.